src/systems/filecoin_blockchain/struct/block/block.id

```diff
@@ -7,25 +7,24 @@ import addr "github.com/filecoin-project/specs/systems/filecoin_vm/actor/address
 import msg "github.com/filecoin-project/specs/systems/filecoin_vm/message"
 
 type BlockCID ipld.CID
-type MessageRoot ipld.CID
-type ReceiptRoot ipld.CID
 type ChainWeight UVarint
 type ChainEpoch UVarint
+type MessageReceipt util.Bytes
 
 // On-chain representation of a block header.
 type BlockHeader struct {
     // Chain linking
     Parents          Tipset
-    Weight           ChainWeight
+    ParentWeight     ChainWeight
     Epoch            ChainEpoch
 
     // Miner Info
     MinerAddress     addr.Address
 
     // State
-    StateTree        st.StateTree
-    Messages         MessageRoot
-    MessageReceipts  ReceiptRoot
+    StateTree        st.StateTree    
```

Comment:
Move the StateTree and MessageRecipts up next to the parents/weight, and document it as the state tree after processing of the parent tipset.

The Messages are proposed messages for this epoch.

Reply:
Move BLSAggregateSig next to Messages